### PR TITLE
Prepare minor release 0.2.0

### PR DESCRIPTION
**Pull request by geek-yang**
- Update changelog
- Update setup.cfg
- Bump version to 0.2.0

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -32,9 +32,8 @@ authors:
     given-names: Jannes
     affiliation: "Vrije Universiteit Amsterdam"
 
-date-released: 20??-MM-DD
-doi: <insert your DOI here>
-version: "0.1.0"
+date-released: 2022-09-01
+version: "version: "0.2.0""
```
Comment (Peter9192) [inline CITATION.cff:36]: There's a double "version" here, is that correct?

Comment (geek-yang) [reply CITATION.cff:36]: Thanks for spotting this. Yep, this is a bug caused by `bump2version`. When I run it, it added these double versions thingy to some files...Had a discussion with Sarah but we don't know why...So I simply delete these double versions. But forgot this one. I will investigate this a bit.

Comment (geek-yang) [reply CITATION.cff]: Quite strange...`bump2version` just not work...When I simply run `bump2version 0.2.0`, it keeps causing double versions and it only gives "0.1.0". It seems `bump2version` fails to recognize the version, although `.bumpversion.cfg` already exists. Also did not find anything useful after some googling. I will ask Stefen tomorrow. For now, I can only change it manually 😢 .

File: CITATION.cff [hunk 2]
```diff
@@ -32,9 +32,8 @@ authors:
     given-names: Jannes
     affiliation: "Vrije Universiteit Amsterdam"
 
-date-released: 20??-MM-DD
-doi: <insert your DOI here>
-version: "0.1.0"
+date-released: 2022-09-01
+version: "0.2.0""
```
Comment (SarahAlidoost) [inline CITATION.cff:36]:
```suggestion
version: "0.2.0"
```

**Review by SarahAlidoost — APPROVED** [review]
@geek-yang looks good to me, just a small fix on the `CITATION.cff`. Also, please submit an issue about the usage of  `bump2version`.

**Comment by geek-yang** [comment]
> @geek-yang looks good to me, just a small fix on the `CITATION.cff`. Also, please submit an issue about the usage of `bump2version`.

Thanks for the feedback. This #98 issue relates to the bug of `bump2version`.

**Pull request merged**